### ISSUE TITLE
hydrological year in 'ggof'

### ISSUE THREAD

**Issue by hzambran** [hzambran/hydroGOF#1]
be able to start the plots in any month different from January.

**Comment by hiztirab** [comment]
I am trying to execute the command for hydroGOF to compare two data sets but getting these error,   
[ Note: 'sim' & 'obs' are not zoo objects => 'ftype' was changed to 'o' ]
Error in xy.coords(x, y, xlabel, ylabel, log) : 
  'x' is a list, but does not have components 'x' and 'y'
In addition: Warning messages:
1: 'rNSE' can not be computed: some elements in 'obs' are zero ! 
2: 'rd' can not be computed: some elements in 'obs' are zero ! 

How i can fix these issues?

**Comment by hzambran** [comment]
Dear hiztirab,

The question you make is not related to this thread ('hydrological year in ´ggof´). Could you create a new DIFFERENT issue and provide a reproducible example of your problem?
So far, it seems that your data are not ´zoo´ objects but ´lists´.

**Comment by hiztirab** [comment]
Yes, i converted my list into zoo and executed the command but this type of warning is still there Warning messages:
1: 'rNSE' can not be computed: some elements in 'obs' are zero ! 
2: 'rd' can not be computed: some elements in 'obs' are zero!

Moreover i tried to modify the "gof" in to "gof.all" by selecting some interested parameters but could not succeed. 
Note, I have followed your threads and codes for all above.

**Comment by hiztirab** [comment]
Actually i want to compare two data sets on different temporal and spatial scale with selected parameters as like (mean, absolute mean error, biase, RMSE, R, R^2, NSE, CSI,)

**Comment by hzambran** [comment]
Could you provide a minimum **reproducible example** to know where the problem is? (in a **new thread!**)
